src/components/cards/CreatePool/PoolType.vue

```diff
@@ -0,0 +1,123 @@
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n';
+
+import { PoolType } from '@/services/balancer/subgraph/types';
+
+import NetworkCard from '../NetworkCard/NetworkCard.vue';
+import usePoolCreation from '@/composables/pools/usePoolCreation';
+
+/**
+ * CONSTANTS
+ */
+const SUPPORTED_POOL_TYPES = [PoolType.Weighted, PoolType.Managed];
+
+/**
+ * COMPOSABLES
+ */
+const { t } = useI18n();
+const { type: poolType, proceed } = usePoolCreation();
+
+/**
+ * COMPUTED
+ */
+const poolTypeDescriptionMap: Partial<Record<PoolType, string>> = {
+  [PoolType.Weighted]: t('createAPool.weightedPoolDescription'),
+  [PoolType.Managed]: t('createAPool.managedPoolDescription')
+};
+
+/**
+ * METHODS
+ */
+function handlePoolTypeSelected(chosenPoolType: PoolType) {
+  poolType.value = chosenPoolType;
+  proceed();
+}
+
+/** VIEW ONLY COMPONENTS */
+const PoolTypeButton = {
+  props: {
+    title: String,
+    text: String
+  },
+  template: /*html*/ `
+    <button class="border bg-white rounded-xl relative p-4 pool-type-button">
+      <BalStack horizontal align='center' class="relative">
+        <div class="rounded-lg pool-type-image border bg-gray-50 w-24 h-24" />
+        <BalStack vertical spacing="xs" align="start">
+          <h5 class="font-bold dark:text-gray-300">
+            {{ title }}
+          </h5>
+          <p class="text-sm text-left">
+            {{ text }}
+          </p>
+        </BalStack>
+        <BalIcon name="chevron-right" class="chevron absolute right-0 text-blue-500" />
+      </BalStack>
+    </button>
+  `
+};
```

Comment:
This is a new thing which I want to get feedback for. 
The thinking here is that I wanted to keep the template pretty clean and readable and to do that, I wanted to have a separate component for this 'PoolTypeButton'. However at the same time I didn't think it needed its own file, its just a dumb view component. So I thought an inline component would be much easier to work with here.

Thoughts?